mission/functions/systems/director/fn_director_init.sqf

```diff
@@ -18,10 +18,12 @@
 */
 
 //Array in format - [Task Object]
-mf_s_activeZones = [];
-mf_s_siegedZones = [];
+mf_s_dir_activeZones = createHashMap;
 mf_s_baseZoneUnlockDistance = 4000;
 
+mf_g_dir_activeZoneNames = [];
+publicVariable "mf_g_dir_activeZoneNames";
+
```

Comment:
will need to test / check this as might be called in other places in our additions.